### PR TITLE
Header footer

### PR DESCRIPTION
**Pull request by jgoley**
Adds basic styles for header and footer

### CODE REVIEW

File: packages/styles/footer.import.styl [hunk 1]
```diff
@@ -0,0 +1,41 @@
+footer
+  position absolute
+  bottom 0
+  width 100%
+  height $footer-height
+  z-index 100
```
Comment (aslagle) [inline packages/styles/footer.import.styl]: maybe we could have some variables for the z-index

Comment (jgoley) [inline packages/styles/footer.import.styl]: I've added three layers above the document base: 
10, 100, 1000 `$bottom-layer $mid-layer $top-layer`

**Comment by aslagle** [comment]
I see a js error from accountsModal:

Exception in template helper: TypeError: Cannot read property 'get' of undefined
    at Object.Template.accountsModal.helpers.state (http://localhost:3000/packages/mantle_controllers.js?e235021c1f0edf861db258ce383b17ca639c4ee8:66:37)
    at bindDataContext (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2880:16)
    at Blaze._wrapCatchingExceptions (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:1651:16)
    at http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2928:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:3476:12)
    at wrapHelper (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2927:27)
    at Object.Spacebars.call (http://localhost:3000/packages/spacebars.js?7bafbe05ec09b6bbb6a3b276537e4995ab298a2f:172:18)
    at http://localhost:3000/packages/mantle_views.js?db5b12318d4c97ab0649485963ffd00927b6407f:50:24
    at Blaze._TemplateWith.wrappedArgFunc (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2795:14)
    at null.<anonymous> (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2599:26)

**Comment by jgoley** [comment]
Oh, I moved the modal template placement outside of the header because of z-index issues...I think, ideally, it needs to be in the main container div.
I'll see if I can fix it.

**Comment by jgoley** [comment]
I tried leaving the modal in the header and resolving the issue by manipulating z-index but doing so causes more conflicts with the basic layout—its due to how we structuring the layout to push the footer down.
Does this work for a solution?: 695e0c9

File: packages/controllers/header.coffee [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Template.header.onCreated ->
+  @state = new ReactiveVar("signIn")
```
Comment (aslagle) [inline packages/controllers/header.coffee]: This is fine, but let's call it accountsState

Comment (jgoley) [inline packages/controllers/header.coffee]: Good call...should I also rename the helper?

Comment (aslagle) [inline packages/controllers/header.coffee]: yeah, sure

**Comment by aslagle** [comment]
Could you also merge master in so it will run the tests?

**Comment by jgoley** [comment]
Are we trusting the results of Travis CI now?

**Comment by aslagle** [comment]
It should be working. @frosario any idea why this build failed? Is it the useraccounts thing again?

**Comment by frosario** [comment]
Looks like the useraccounts bug again:
https://travis-ci.org/ecohealthalliance/mantle/builds/70765336#L190

I see `FAILURE` on the next line after it, so that tells me cucumber failed because of it.

**Comment by aslagle** [comment]
Any idea why it would be failing again on this branch?

**Comment by frosario** [comment]
Rerunning the same build to see if we receive the same result again.

**Comment by frosario** [comment]
Looks like it passed this time. Not sure sure why exactly. Considering these are all automated, and non-interactive tests, it makes me think there may be a race condition going on with selenium.

**Comment by aslagle** [comment]
This pr looks good. We'll have to keep working on travis until it's more stable.

**Pull request merged**